build.xml

```diff
@@ -102,6 +102,7 @@
   <target name="spldoctoolkits">
      <exec executable="${streams.install}/bin/spl-make-doc">
         <arg value="--include-composite-operator-diagram"/>
+        <arg value="--include-all"/>
```

Comment:
Not sure this should be here, what exactly did you want to be included because it looks like this includes 'excluded' items as well, which (if they exist) are intentionally excluded from the api of the toolkit?